[2023-03-23 15:48:31,684] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: multi_attack_dag.metadata manual__2023-03-23T15:48:21.866946+00:00 [queued]>
[2023-03-23 15:48:31,741] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: multi_attack_dag.metadata manual__2023-03-23T15:48:21.866946+00:00 [queued]>
[2023-03-23 15:48:31,743] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-03-23 15:48:31,746] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2023-03-23 15:48:31,747] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-03-23 15:48:31,853] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): metadata> on 2023-03-23 15:48:21.866946+00:00
[2023-03-23 15:48:31,872] {standard_task_runner.py:52} INFO - Started process 13919 to run task
[2023-03-23 15:48:31,896] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'multi_attack_dag', 'metadata', 'manual__2023-03-23T15:48:21.866946+00:00', '--job-id', '3569', '--raw', '--subdir', 'DAGS_FOLDER/attack_dag.py', '--cfg-path', '/tmp/tmp5coron3g', '--error-file', '/tmp/tmp9rsthsre']
[2023-03-23 15:48:31,904] {standard_task_runner.py:80} INFO - Job 3569: Subtask metadata
[2023-03-23 15:48:32,243] {task_command.py:369} INFO - Running <TaskInstance: multi_attack_dag.metadata manual__2023-03-23T15:48:21.866946+00:00 [running]> on host 6447f1dacd32
[2023-03-23 15:48:32,621] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=multi_attack_dag
AIRFLOW_CTX_TASK_ID=metadata
AIRFLOW_CTX_EXECUTION_DATE=2023-03-23T15:48:21.866946+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-23T15:48:21.866946+00:00
[2023-03-23 15:48:32,660] {transport.py:157} INFO - Attempting refresh to obtain initial access_token
[2023-03-23 15:48:32,670] {client.py:777} INFO - Refreshing access_token
[2023-03-23 15:48:33,942] {python.py:173} INFO - Done. Returned value was: None
[2023-03-23 15:48:33,987] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=multi_attack_dag, task_id=metadata, execution_date=20230323T154821, start_date=20230323T154831, end_date=20230323T154833
[2023-03-23 15:48:34,050] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-03-23 15:48:34,193] {local_task_job.py:273} INFO - 3 downstream tasks scheduled from follow-on schedule check
